### ISSUE TITLE
[Accessibility] Panel: Cannot Access 'X' (Close) Button in Scan Mode

### ISSUE THREAD

**Issue by HalcyonPride** [OfficeDev/office-ui-fabric-react#5608]
### Bug Report 
 
- __Package version(s)__: Latest, on https://developer.microsoft.com/en-us/fabric
- __Browser and OS versions__: Edge/Narrator, on Windows 10 Enterprise; Version 1803; OS 17134.165
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: **High**
 
Products/sites affected: All
 
#### Describe the issue:

Users cannot navigate to the 'X' button on top of Panels in Scan Mode on Narrator.
- Turn on Scan Mode in Narrator. 
- Go to https://developer.microsoft.com/en-us/fabric#/components/panel
- Open any Panel in the examples.
- Using the arrow keys, in Scan Mode, navigate to the 'X' button on top of the Panel.

#### Actual behavior:
 
When navigating to the 'X' button, focus falls out of the panel and back into the main page instead.
 
#### Expected behavior:
 
User can navigate to the 'X' button in Scan Mode.
 
### If applicable, please provide a codepen repro:

Can repro here: https://developer.microsoft.com/en-us/fabric#/components/panel

**Comment by betrue-final-final** [comment]
Yes, it seems that even though you start on the close X, you can't get back to it. Tabbing should cycle from the Cancel button back up to the close X, and then back again. Also focus should be trapped.

**Comment by HalcyonPride** [comment]
N.B. You **can Tab** to the 'X' Button, but you **cannot use Arrow Keys** to get to it in Scan Mode.

**Comment by philipzloh** [comment]
Please file a bug with Narrator. This is purely a Narrator issue with how Scan Mode works. Scan Mode works poorly when it is expected to seamlessly move between "text" and "items".

**Comment by HalcyonPride** [comment]
@MaxLustig confirmed that he's tried this Bug with a newer version of Narrator and it didn't repro, thus we can close it.

**Comment by AkshanshSrivastava03** [comment]
@HalcyonPride/ @MaxLustig  - Is there some way we can test the behavior in newer version of narrator? We are facing a11y issues with each instance of IconButton used in the product (with scan mode) and would love to get some hands-on the fix, if possible.

**Comment by MaxLustig** [comment]
I tried it out on an RS5 build of Narrator and confirmed that it reads the close button in scan mode.

**Comment by HalcyonPride** [comment]
Please see my email.
